src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
@davidfowl, @Eilon, these are internal types in the Universe repo. Are you going to make them public?

Microsoft.AspNet.Authentication
        Constants in Microsoft.AspNet.Authentication.dll
        HttpContextExtensions in Microsoft.AspNet.Authentication.dll
        Resources in Microsoft.AspNet.Authentication.dll

Microsoft.AspNet.Authentication.Cookies
        CookieAuthenticationHandler in Microsoft.AspNet.Authentication.Cookies.dll
        DefaultBehavior in Microsoft.AspNet.Authentication.Cookies.dll
        Resources in Microsoft.AspNet.Authentication.Cookies.dll

Microsoft.AspNet.Authentication.Cookies.Infrastructure
        Constants in Microsoft.AspNet.Authentication.Cookies.dll

Microsoft.AspNet.Authentication.Facebook
        FacebookAuthenticationHandler in Microsoft.AspNet.Authentication.Facebook.dll
        Resources in Microsoft.AspNet.Authentication.Facebook.dll

Microsoft.AspNet.Authentication.Google
        GoogleAuthenticationHandler in Microsoft.AspNet.Authentication.Google.dll
        Resources in Microsoft.AspNet.Authentication.Google.dll

Microsoft.AspNet.Authentication.MicrosoftAccount
        MicrosoftAccountAuthenticationHandler in Microsoft.AspNet.Authentication.MicrosoftAccount.dll
        Resources in Microsoft.AspNet.Authentication.MicrosoftAccount.dll

Microsoft.AspNet.Authentication.OAuth
        Resources in Microsoft.AspNet.Authentication.OAuth.dll

Microsoft.AspNet.Authentication.OAuthBearer
        Resources in Microsoft.AspNet.Authentication.OAuthBearer.dll

Microsoft.AspNet.Authentication.OpenIdConnect
        Resources in Microsoft.AspNet.Authentication.OpenIdConnect.dll

Microsoft.AspNet.Authentication.Twitter
        Resources in Microsoft.AspNet.Authentication.Twitter.dll
        TwitterAuthenticationHandler in Microsoft.AspNet.Authentication.Twitter.dll

Microsoft.AspNet.Authorization
        Resources in Microsoft.AspNet.Authorization.dll

Microsoft.AspNet.Cors.Core
        Resources in Microsoft.AspNet.Cors.Core.dll

Microsoft.AspNet.Cryptography
        Constants in Microsoft.AspNet.Cryptography.Internal.dll
        CryptoUtil in Microsoft.AspNet.Cryptography.Internal.dll
        DATA_BLOB in Microsoft.AspNet.Cryptography.Internal.dll
        UnsafeBufferUtil in Microsoft.AspNet.Cryptography.Internal.dll
        UnsafeNativeMethods in Microsoft.AspNet.Cryptography.Internal.dll
        WeakReferenceHelpers in Microsoft.AspNet.Cryptography.Internal.dll

Microsoft.AspNet.Cryptography.Cng
        BCRYPT_AUTHENTICATED_CIPHER_MODE_INFO in Microsoft.AspNet.Cryptography.Internal.dll
        BCRYPT_KEY_LENGTHS_STRUCT in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptBuffer in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptBufferDesc in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptEncryptFlags in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptGenRandomFlags in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptKeyDerivationBufferType in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptUtil in Microsoft.AspNet.Cryptography.Internal.dll
        CachedAlgorithmHandles in Microsoft.AspNet.Cryptography.Internal.dll
        NCryptEncryptFlags in Microsoft.AspNet.Cryptography.Internal.dll
        OSVersionUtil in Microsoft.AspNet.Cryptography.Internal.dll

Microsoft.AspNet.Cryptography.Internal
        Resources in Microsoft.AspNet.Cryptography.Internal.dll

Microsoft.AspNet.Cryptography.KeyDerivation.PBKDF2
        IPbkdf2Provider in Microsoft.AspNet.Cryptography.KeyDerivation.dll
        ManagedPbkdf2Provider in Microsoft.AspNet.Cryptography.KeyDerivation.dll
        Pbkdf2Util in Microsoft.AspNet.Cryptography.KeyDerivation.dll
        Win7Pbkdf2Provider in Microsoft.AspNet.Cryptography.KeyDerivation.dll
        Win8Pbkdf2Provider in Microsoft.AspNet.Cryptography.KeyDerivation.dll

Microsoft.AspNet.Cryptography.SafeHandles
        BCryptAlgorithmHandle in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptHandle in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptHashHandle in Microsoft.AspNet.Cryptography.Internal.dll
        BCryptKeyHandle in Microsoft.AspNet.Cryptography.Internal.dll
        LocalAllocHandle in Microsoft.AspNet.Cryptography.Internal.dll
        NCryptDescriptorHandle in Microsoft.AspNet.Cryptography.Internal.dll
        SafeLibraryHandle in Microsoft.AspNet.Cryptography.Internal.dll
        SecureLocalAllocHandle in Microsoft.AspNet.Cryptography.Internal.dll

Microsoft.AspNet.DataProtection
        ActivatorExtensions in Microsoft.AspNet.DataProtection.dll
        ApplyPolicyAttribute in Microsoft.AspNet.DataProtection.dll
        ArraySegmentExtensions in Microsoft.AspNet.DataProtection.dll
        BitHelpers in Microsoft.AspNet.DataProtection.dll
        CryptoUtil in Microsoft.AspNet.DataProtection.Abstractions.dll
        DataProtectionProviderFactory in Microsoft.AspNet.DataProtection.dll
        EncodingUtil in Microsoft.AspNet.DataProtection.dll
        Error in Microsoft.AspNet.DataProtection.dll
        ExceptionExtensions in Microsoft.AspNet.DataProtection.dll
        IActivator in Microsoft.AspNet.DataProtection.dll
        MemoryProtection in Microsoft.AspNet.DataProtection.dll
        RegistryPolicyResolver in Microsoft.AspNet.DataProtection.dll
        Resources in Microsoft.AspNet.DataProtection.dll
        TimeLimitedDataProtector in Microsoft.AspNet.DataProtection.Extensions.dll
        TypeExtensions in Microsoft.AspNet.DataProtection.dll
        WebEncoders in Microsoft.AspNet.DataProtection.Abstractions.dll
        XmlConstants in Microsoft.AspNet.DataProtection.dll
        XmlExtensions in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.Abstractions
        Resources in Microsoft.AspNet.DataProtection.Abstractions.dll

Microsoft.AspNet.DataProtection.AuthenticatedEncryption
        AlgorithmAssert in Microsoft.AspNet.DataProtection.dll
        AuthenticatedEncryptorExtensions in Microsoft.AspNet.DataProtection.dll
        IInternalAuthenticatedEncryptionOptions in Microsoft.AspNet.DataProtection.dll
        IOptimizedAuthenticatedEncryptor in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.AuthenticatedEncryption.ConfigurationModel
        ConfigurationCommon in Microsoft.AspNet.DataProtection.dll
        IInternalAuthenticatedEncryptorConfiguration in Microsoft.AspNet.DataProtection.dll
        SecretExtensions in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.Cng
        BCryptGenRandomImpl in Microsoft.AspNet.DataProtection.dll
        CbcAuthenticatedEncryptor in Microsoft.AspNet.DataProtection.dll
        CngAuthenticatedEncryptorBase in Microsoft.AspNet.DataProtection.dll
        DpapiSecretSerializerHelper in Microsoft.AspNet.DataProtection.dll
        GcmAuthenticatedEncryptor in Microsoft.AspNet.DataProtection.dll
        IBCryptGenRandom in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.Extensions
        Resources in Microsoft.AspNet.DataProtection.Extensions.dll

Microsoft.AspNet.DataProtection.KeyManagement
        CacheableKeyRing in Microsoft.AspNet.DataProtection.dll
        DefaultKeyResolution in Microsoft.AspNet.DataProtection.dll
        DefaultKeyResolver in Microsoft.AspNet.DataProtection.dll
        DefaultKeyServices in Microsoft.AspNet.DataProtection.dll
        DeferredKey in Microsoft.AspNet.DataProtection.dll
        ICacheableKeyRingProvider in Microsoft.AspNet.DataProtection.dll
        IDefaultKeyResolver in Microsoft.AspNet.DataProtection.dll
        IDefaultKeyServices in Microsoft.AspNet.DataProtection.dll
        IInternalXmlKeyManager in Microsoft.AspNet.DataProtection.dll
        IKeyRing in Microsoft.AspNet.DataProtection.dll
        IKeyRingProvider in Microsoft.AspNet.DataProtection.dll
        Key in Microsoft.AspNet.DataProtection.dll
        KeyBase in Microsoft.AspNet.DataProtection.dll
        KeyEscrowServiceProviderExtensions in Microsoft.AspNet.DataProtection.dll
        KeyExtensions in Microsoft.AspNet.DataProtection.dll
        KeyRing in Microsoft.AspNet.DataProtection.dll
        KeyRingBasedDataProtectionProvider in Microsoft.AspNet.DataProtection.dll
        KeyRingBasedDataProtector in Microsoft.AspNet.DataProtection.dll
        KeyRingProvider in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.Managed
        HashAlgorithmExtensions in Microsoft.AspNet.DataProtection.dll
        IManagedGenRandom in Microsoft.AspNet.DataProtection.dll
        ManagedAuthenticatedEncryptor in Microsoft.AspNet.DataProtection.dll
        ManagedGenRandomImpl in Microsoft.AspNet.DataProtection.dll
        SymmetricAlgorithmExtensions in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.Repositories
        EphemeralXmlRepository in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.SP800_108
        ISP800_108_CTR_HMACSHA512Provider in Microsoft.AspNet.DataProtection.dll
        ManagedSP800_108_CTR_HMACSHA512 in Microsoft.AspNet.DataProtection.dll
        SP800_108_CTR_HMACSHA512Extensions in Microsoft.AspNet.DataProtection.dll
        SP800_108_CTR_HMACSHA512Util in Microsoft.AspNet.DataProtection.dll
        Win7SP800_108_CTR_HMACSHA512Provider in Microsoft.AspNet.DataProtection.dll
        Win8SP800_108_CTR_HMACSHA512Provider in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.DataProtection.SystemWeb
        Resources in Microsoft.AspNet.DataProtection.SystemWeb.dll

Microsoft.AspNet.DataProtection.XmlEncryption
        EncryptedXmlDecryptor in Microsoft.AspNet.DataProtection.dll
        IInternalCertificateXmlEncryptor in Microsoft.AspNet.DataProtection.dll
        IInternalEncryptedXmlDecryptor in Microsoft.AspNet.DataProtection.dll
        XmlEncryptionExtensions in Microsoft.AspNet.DataProtection.dll

Microsoft.AspNet.Diagnostics
        Constants in Microsoft.AspNet.Diagnostics.dll
        Resources in Microsoft.AspNet.Diagnostics.dll

Microsoft.AspNet.Diagnostics.Entity
        Strings in Microsoft.AspNet.Diagnostics.Entity.dll

Microsoft.AspNet.Diagnostics.Entity.Utilities
        Check in Microsoft.AspNet.Diagnostics.Entity.dll

Microsoft.AspNet.FileProviders
        EnumerableDirectoryContents in Microsoft.AspNet.FileProviders.Embedded.dll
        FileChangeTrigger in Microsoft.AspNet.FileProviders.Physical.dll
        NoopTrigger in Microsoft.AspNet.FileProviders.Embedded.dll
        NotFoundDirectoryContents in Microsoft.AspNet.FileProviders.Embedded.dll
        NotFoundFileInfo in Microsoft.AspNet.FileProviders.Embedded.dll
        PhysicalFilesWatcher in Microsoft.AspNet.FileProviders.Physical.dll

Microsoft.AspNet.Http.Extensions
        Resources in Microsoft.AspNet.Http.Extensions.dll

Microsoft.AspNet.Http.Internal
        Constants in Microsoft.AspNet.Http.dll
        HeaderSegment in Microsoft.AspNet.Http.dll
        HeaderSegmentCollection in Microsoft.AspNet.Http.dll
        ParsingHelpers in Microsoft.AspNet.Http.dll
        ReferenceReadStream in Microsoft.AspNet.Http.dll
        StringSegment in Microsoft.AspNet.Http.dll

Microsoft.AspNet.Identity
        Resources in Microsoft.AspNet.Identity.dll
        Rfc6238AuthenticationService in Microsoft.AspNet.Identity.dll
        StreamExtensions in Microsoft.AspNet.Identity.dll

Microsoft.AspNet.Identity.EntityFramework
        Resources in Microsoft.AspNet.Identity.EntityFramework.dll

Microsoft.AspNet.JsonPatch
        Resources in Microsoft.AspNet.JsonPatch.dll

Microsoft.AspNet.JsonPatch.Helpers
        ConversionResult in Microsoft.AspNet.JsonPatch.dll
        ExpressionHelpers in Microsoft.AspNet.JsonPatch.dll

Microsoft.AspNet.Mvc
        AntiForgeryContext in Microsoft.AspNet.Mvc.Core.dll
        AntiForgeryToken in Microsoft.AspNet.Mvc.Core.dll
        AntiForgeryTokenProvider in Microsoft.AspNet.Mvc.Core.dll
        AntiForgeryTokenSerializer in Microsoft.AspNet.Mvc.Core.dll
        AntiForgeryTokenStore in Microsoft.AspNet.Mvc.Core.dll
        AntiForgeryWorker in Microsoft.AspNet.Mvc.Core.dll
        BinaryBlob in Microsoft.AspNet.Mvc.Core.dll
        Encodings in Microsoft.AspNet.Mvc.Abstractions.dll
        IAntiForgeryTokenGenerator in Microsoft.AspNet.Mvc.Core.dll
        IAntiForgeryTokenSerializer in Microsoft.AspNet.Mvc.Core.dll
        IAntiForgeryTokenStore in Microsoft.AspNet.Mvc.Core.dll
        IAntiForgeryTokenValidator in Microsoft.AspNet.Mvc.Core.dll
        ResolveFilePathResult in Microsoft.AspNet.Mvc.Core.dll
        Resources in Microsoft.AspNet.Mvc.dll
        TaskHelper in Microsoft.AspNet.Mvc.Abstractions.dll
        TypeHelper in Microsoft.AspNet.Mvc.Abstractions.dll

Microsoft.AspNet.Mvc.Abstractions
        Resources in Microsoft.AspNet.Mvc.Abstractions.dll

Microsoft.AspNet.Mvc.Core
        Resources in Microsoft.AspNet.Mvc.Core.dll

Microsoft.AspNet.Mvc.Core.Internal
        SerializerSettingsProvider in Microsoft.AspNet.Mvc.Core.dll

Microsoft.AspNet.Mvc.ModelBinding
        ElementalValueProvider in Microsoft.AspNet.Mvc.Core.dll

Microsoft.AspNet.Mvc.ModelBinding.Validation
        ReferenceEqualityComparer in Microsoft.AspNet.Mvc.Core.dll

Microsoft.AspNet.Mvc.Razor
        Resources in Microsoft.AspNet.Mvc.Razor.dll

Microsoft.AspNet.Mvc.Razor.Host
        Resources in Microsoft.AspNet.Mvc.Razor.Host.dll

Microsoft.AspNet.Mvc.Razor.Internal
        Crc32 in Microsoft.AspNet.Mvc.Razor.dll

Microsoft.AspNet.Mvc.Rendering
        HtmlAttributePropertyHelper in Microsoft.AspNet.Mvc.Core.dll
        ValidationHelpers in Microsoft.AspNet.Mvc.Core.dll

Microsoft.AspNet.Mvc.TagHelpers
        Resources in Microsoft.AspNet.Mvc.TagHelpers.dll

Microsoft.AspNet.Mvc.WebApiCompatShim
        Resources in Microsoft.AspNet.Mvc.WebApiCompatShim.dll

Microsoft.AspNet.Mvc.Xml
        Resources in Microsoft.AspNet.Mvc.Xml.dll

Microsoft.AspNet.Owin
        OwinConstants in Microsoft.AspNet.Owin.dll
        Utilities in Microsoft.AspNet.Owin.dll

Microsoft.AspNet.Razor
        CommonResources in Microsoft.AspNet.Razor.dll
        RazorDebugHelpers in Microsoft.AspNet.Razor.dll
        RazorResources in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Editor
        BackgroundParser in Microsoft.AspNet.Razor.dll
        RazorEditorTrace in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Parser
        ConditionalAttributeCollapser in Microsoft.AspNet.Razor.dll
        MarkupCollapser in Microsoft.AspNet.Razor.dll
        MarkupRewriter in Microsoft.AspNet.Razor.dll
        TextReaderExtensions in Microsoft.AspNet.Razor.dll
        WhiteSpaceRewriter in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Parser.SyntaxTree
        EquivalenceComparer in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Runtime
        Resources in Microsoft.AspNet.Razor.Runtime.dll

Microsoft.AspNet.Razor.Text
        LineTrackingStringBuffer in Microsoft.AspNet.Razor.dll
        TextExtensions in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Tokenizer
        CSharpKeywordDetector in Microsoft.AspNet.Razor.dll
        XmlHelpers in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Razor.Utils
        CharUtils in Microsoft.AspNet.Razor.dll
        DisposableAction in Microsoft.AspNet.Razor.dll
        EnumeratorExtensions in Microsoft.AspNet.Razor.dll
        EnumUtil in Microsoft.AspNet.Razor.dll

Microsoft.AspNet.Routing
        Resources in Microsoft.AspNet.Routing.dll

Microsoft.AspNet.Routing.Logging
        StringBuilderHelpers in Microsoft.AspNet.Routing.dll

Microsoft.AspNet.Server.Kestrel.Http
        FrameDuplexStream in Microsoft.AspNet.Server.Kestrel.dll
        FrameResponseStream in Microsoft.AspNet.Server.Kestrel.dll

Microsoft.AspNet.SignalR
        AssemblyExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        RequestExtensions in Microsoft.AspNet.SignalR.Server.dll
        Resources in Microsoft.AspNet.SignalR.Server.dll
        ResponseExtensions in Microsoft.AspNet.SignalR.Server.dll
        TaskAsyncHelper in Microsoft.AspNet.SignalR.Messaging.dll
        TaskAwaiterHelper in Microsoft.AspNet.SignalR.Messaging.dll

Microsoft.AspNet.SignalR.Configuration
        TransportOptionsExtensions in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.SignalR.Hubs
        HubContext in Microsoft.AspNet.SignalR.Server.dll
        HubContext`2 in Microsoft.AspNet.SignalR.Server.dll
        HubInvocationProgress in Microsoft.AspNet.SignalR.Server.dll
        HubInvocationProgress`1 in Microsoft.AspNet.SignalR.Server.dll
        HubInvokerContext in Microsoft.AspNet.SignalR.Server.dll
        HubMethodDispatcher in Microsoft.AspNet.SignalR.Server.dll
        HubOutgoingInvokerContext in Microsoft.AspNet.SignalR.Server.dll
        HubOutgoingInvokerContextExtensions in Microsoft.AspNet.SignalR.Server.dll
        HubPipeline in Microsoft.AspNet.SignalR.Server.dll
        HubRequestParser in Microsoft.AspNet.SignalR.Server.dll
        HubTypeExtensions in Microsoft.AspNet.SignalR.Server.dll
        NullClientProxy in Microsoft.AspNet.SignalR.Server.dll
        TypedClientBuilder`1 in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.SignalR.Infrastructure
        AckSubscriber in Microsoft.AspNet.SignalR.Server.dll
        ByteBuffer in Microsoft.AspNet.SignalR.Server.dll
        CancellationTokenExtensions in Microsoft.AspNet.SignalR.Server.dll
        DiffPair`1 in Microsoft.AspNet.SignalR.Server.dll
        DiffSet`1 in Microsoft.AspNet.SignalR.Server.dll
        DisposableAction in Microsoft.AspNet.SignalR.Messaging.dll
        Disposer in Microsoft.AspNet.SignalR.Messaging.dll
        ExceptionsExtensions in Microsoft.AspNet.SignalR.Messaging.dll
        ListHelper`1 in Microsoft.AspNet.SignalR.Messaging.dll
        MonoUtility in Microsoft.AspNet.SignalR.Server.dll
        PersistentConnectionContext in Microsoft.AspNet.SignalR.Server.dll
        PrefixHelper in Microsoft.AspNet.SignalR.Server.dll
        SafeCancellationTokenSource in Microsoft.AspNet.SignalR.Server.dll
        SafeSet`1 in Microsoft.AspNet.SignalR.Server.dll
        SipHashBasedStringEqualityComparer in Microsoft.AspNet.SignalR.Messaging.dll
        StreamExtensions in Microsoft.AspNet.SignalR.Server.dll
        TaskQueue in Microsoft.AspNet.SignalR.Messaging.dll
        UrlDecoder in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.SignalR.Json
        JRawValue in Microsoft.AspNet.SignalR.Server.dll
        SipHashBasedDictionaryConverter in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.SignalR.Messaging
        Cursor in Microsoft.AspNet.SignalR.Messaging.dll
        DefaultSubscription in Microsoft.AspNet.SignalR.Messaging.dll
        Resources in Microsoft.AspNet.SignalR.Messaging.dll
        ScaleoutStream in Microsoft.AspNet.SignalR.Messaging.dll
        ScaleoutStreamManager in Microsoft.AspNet.SignalR.Messaging.dll
        TopicState in Microsoft.AspNet.SignalR.Messaging.dll
        Volatile in Microsoft.AspNet.SignalR.Messaging.dll

Microsoft.AspNet.SignalR.Redis
        Resources in Microsoft.AspNet.SignalR.Redis.dll

Microsoft.AspNet.SignalR.SqlServer
        DbDataReaderExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        DbOperation in Microsoft.AspNet.SignalR.SqlServer.dll
        DbParameterExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        DbProviderFactoryAdapter in Microsoft.AspNet.SignalR.SqlServer.dll
        DbProviderFactoryExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        IDataRecordExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        IDbCommandExtensions in Microsoft.AspNet.SignalR.SqlServer.dll
        ObservableDbOperation in Microsoft.AspNet.SignalR.SqlServer.dll
        Resources in Microsoft.AspNet.SignalR.SqlServer.dll
        SqlInstaller in Microsoft.AspNet.SignalR.SqlServer.dll
        SqlReceiver in Microsoft.AspNet.SignalR.SqlServer.dll
        SqlSender in Microsoft.AspNet.SignalR.SqlServer.dll
        SqlStream in Microsoft.AspNet.SignalR.SqlServer.dll

Microsoft.AspNet.SignalR.Transports
        HttpRequestLifeTime in Microsoft.AspNet.SignalR.Server.dll
        TransportConnectionExtensions in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.SignalR.WebSockets
        IWebSocket in Microsoft.AspNet.SignalR.Server.dll
        WebSocketMessage in Microsoft.AspNet.SignalR.Server.dll
        WebSocketMessageReader in Microsoft.AspNet.SignalR.Server.dll

Microsoft.AspNet.StaticFiles
        Constants in Microsoft.AspNet.StaticFiles.dll
        Helpers in Microsoft.AspNet.StaticFiles.dll
        Resources in Microsoft.AspNet.StaticFiles.dll
        StaticFileContext in Microsoft.AspNet.StaticFiles.dll
        StreamCopyOperation in Microsoft.AspNet.StaticFiles.dll

Microsoft.AspNet.StaticFiles.Infrastructure
        RangeHelpers in Microsoft.AspNet.StaticFiles.dll

Microsoft.AspNet.TestHost
        RequestFeature in Microsoft.AspNet.TestHost.dll
        ResponseFeature in Microsoft.AspNet.TestHost.dll
        ResponseStream in Microsoft.AspNet.TestHost.dll

Microsoft.AspNet.Testing.xunit
        ConditionalAttributeDiscoverer in Microsoft.AspNet.Testing.dll
        SkipReasonTestCase in Microsoft.AspNet.Testing.dll

Microsoft.AspNet.WebUtilities
        BufferedReadStream in Microsoft.AspNet.WebUtilities.dll
        MultipartReaderStream in Microsoft.AspNet.WebUtilities.dll

Microsoft.Data.Entity.Relational.Utilities
        ValidatedNotNullAttribute in Microsoft.AspNet.Diagnostics.Entity.dll

Microsoft.Data.Entity.Utilities
        Check in EntityFramework.Commands.dll

Microsoft.Framework.Caching.Memory
        CacheEntry in Microsoft.Framework.Caching.Memory.dll
        EntryLinkHelpers in Microsoft.Framework.Caching.Memory.dll

Microsoft.Framework.Caching.Redis
        RedisExtensions in Microsoft.Framework.Caching.Redis.dll

Microsoft.Framework.CodeGeneration
        CommonUtilities in Microsoft.Framework.CodeGeneration.dll
        CompilationResult in Microsoft.Framework.CodeGeneration.dll
        DefaultFileSystem in Microsoft.Framework.CodeGeneration.Core.dll
        ExceptionUtilities in Microsoft.Framework.CodeGeneration.dll
        IFileSystem in Microsoft.Framework.CodeGeneration.Core.dll
        MemberInfoExtensions in Microsoft.Framework.CodeGeneration.Core.dll
        NotNullAttribute in Microsoft.Framework.CodeGeneration.dll
        PlatformHelper in Microsoft.Framework.CodeGeneration.dll
        ServiceProvider in Microsoft.Framework.CodeGeneration.dll
        TemplateFoldersUtilities in Microsoft.Framework.CodeGeneration.dll
        TypeExtensions in Microsoft.Framework.CodeGeneration.dll

Microsoft.Framework.CodeGeneration.EntityFramework
        TypeUtilities in Microsoft.Framework.CodeGeneration.EntityFramework.dll

Microsoft.Framework.CodeGeneration.Templating
        RazorTemplatingHost in Microsoft.Framework.CodeGeneration.Templating.dll

Microsoft.Framework.CodeGenerators.Mvc
        Constants in Microsoft.Framework.CodeGenerators.Mvc.dll
        PackageConstants in Microsoft.Framework.CodeGenerators.Mvc.dll
        StandardDependencies in Microsoft.Framework.CodeGenerators.Mvc.dll
        StringUtil in Microsoft.Framework.CodeGenerators.Mvc.dll
        TypeUtilities in Microsoft.Framework.CodeGenerators.Mvc.dll
        ValidationUtil in Microsoft.Framework.CodeGenerators.Mvc.dll

Microsoft.Framework.Configuration
        EncryptedXmlDocumentDecryptor in Microsoft.Framework.Configuration.Xml.dll
        Resources in Microsoft.Framework.Configuration.dll
        XmlDocumentDecryptor in Microsoft.Framework.Configuration.Xml.dll

Microsoft.Framework.Configuration.Binder
        Resources in Microsoft.Framework.Configuration.Binder.dll

Microsoft.Framework.Configuration.CommandLine
        Resources in Microsoft.Framework.Configuration.CommandLine.dll

Microsoft.Framework.Configuration.Ini
        Resources in Microsoft.Framework.Configuration.Ini.dll

Microsoft.Framework.Configuration.Json
        JsonConfigurationFileParser in Microsoft.Framework.Configuration.Json.dll
        Resources in Microsoft.Framework.Configuration.Json.dll

Microsoft.Framework.Configuration.UserSecrets
        Resources in Microsoft.Framework.Configuration.UserSecrets.dll

Microsoft.Framework.Configuration.Xml
        Resources in Microsoft.Framework.Configuration.Xml.dll

Microsoft.Framework.DependencyInjection
        DataProtectionServiceDescriptors in Microsoft.AspNet.DataProtection.dll
        Resources in Microsoft.Framework.DependencyInjection.dll
        ServiceProvider in Microsoft.Framework.DependencyInjection.dll

Microsoft.Framework.DependencyInjection.Abstractions
        Resources in Microsoft.Framework.DependencyInjection.Abstractions.dll

Microsoft.Framework.DependencyInjection.Ninject
        NinjectServiceProvider in Microsoft.Framework.DependencyInjection.Ninject.dll
        NinjectServiceScopeFactory in Microsoft.Framework.DependencyInjection.Ninject.dll
        ScopeParameter in Microsoft.Framework.DependencyInjection.Ninject.dll
        ServiceProviderNinjectModule in Microsoft.Framework.DependencyInjection.Ninject.dll

Microsoft.Framework.DependencyInjection.ServiceLookup
        ClosedIEnumerableService in Microsoft.Framework.DependencyInjection.dll
        FactoryService in Microsoft.Framework.DependencyInjection.dll
        GenericService in Microsoft.Framework.DependencyInjection.dll
        IGenericService in Microsoft.Framework.DependencyInjection.dll
        InstanceService in Microsoft.Framework.DependencyInjection.dll
        IService in Microsoft.Framework.DependencyInjection.dll
        IServiceCallSite in Microsoft.Framework.DependencyInjection.dll
        OpenIEnumerableService in Microsoft.Framework.DependencyInjection.dll
        Service in Microsoft.Framework.DependencyInjection.dll
        ServiceEntry in Microsoft.Framework.DependencyInjection.dll
        ServiceProviderService in Microsoft.Framework.DependencyInjection.dll
        ServiceScope in Microsoft.Framework.DependencyInjection.dll
        ServiceScopeFactory in Microsoft.Framework.DependencyInjection.dll
        ServiceScopeService in Microsoft.Framework.DependencyInjection.dll
        ServiceTable in Microsoft.Framework.DependencyInjection.dll

Microsoft.Framework.Internal
        ActivatorUtilities in Microsoft.AspNet.Http.Abstractions.dll
        BufferEntryCollection in Microsoft.AspNet.Mvc.Core.dll
        ClosedGenericMatcher in Microsoft.AspNet.Mvc.Core.dll
        CopyOnWriteDictionary`2 in Microsoft.AspNet.Mvc.Abstractions.dll
        NotNullAttribute in Microsoft.Framework.Caching.Abstractions.dll
        ObjectFactory in Microsoft.AspNet.Http.Abstractions.dll
        PropertyActivator`1 in Microsoft.AspNet.Mvc.Core.dll
        PropertyHelper in Microsoft.AspNet.JsonPatch.dll
        TypeNameHelper in Microsoft.Framework.Logging.Abstractions.dll

Microsoft.Framework.Logging
        Logger in Microsoft.Framework.Logging.dll
        LoggingExtensions in Microsoft.AspNet.DataProtection.dll
        TraceSourceLogger in Microsoft.Framework.Logging.TraceSource.dll

Microsoft.Framework.OptionsModel
        OptionsConstants in Microsoft.Framework.OptionsModel.dll
        Resources in Microsoft.Framework.OptionsModel.dll

Microsoft.Framework.Runtime
        Constants in Microsoft.Framework.TestHost.dll
        EnvironmentNames in Microsoft.Framework.TestHost.dll
        Logger in Microsoft.Framework.TestHost.dll
        PlatformHelper in Microsoft.Framework.TestHost.dll

Microsoft.Framework.Runtime.Common
        EntryPointExecutor in Microsoft.Framework.TestHost.dll

Microsoft.Framework.Runtime.Common.CommandLine
        AnsiConsole in EntityFramework.Commands.dll
        CommandArgument in EntityFramework.Commands.dll
        CommandLineApplication in EntityFramework.Commands.dll
        CommandOption in EntityFramework.Commands.dll
        CommandOptionType in EntityFramework.Commands.dll

Microsoft.Framework.Runtime.Common.DependencyInjection
        ActivatorUtilities in Microsoft.Framework.TestHost.dll
        ServiceProvider in Microsoft.Framework.TestHost.dll

Microsoft.Framework.Runtime.Common.Impl
        FrameworkNames in Microsoft.Framework.TestHost.dll

Microsoft.Framework.TestHost
        TestServices in Microsoft.Framework.TestHost.dll

Microsoft.Framework.WebEncoders
        AllowedCharsBitmap in Microsoft.Framework.WebEncoders.Core.dll
        EncoderCommon in Microsoft.Framework.WebEncoders.Core.dll
        HexUtil in Microsoft.Framework.WebEncoders.Core.dll
        UnicodeEncoderBase in Microsoft.Framework.WebEncoders.Core.dll
        UnicodeHelpers in Microsoft.Framework.WebEncoders.Core.dll

Microsoft.Net.Http.Headers
        BaseHeaderParser`1 in Microsoft.Net.Http.Headers.dll
        CookieHeaderParser in Microsoft.Net.Http.Headers.dll
        GenericHeaderParser`1 in Microsoft.Net.Http.Headers.dll
        HttpHeaderParser`1 in Microsoft.Net.Http.Headers.dll
        HttpParseResult in Microsoft.Net.Http.Headers.dll
        HttpRuleParser in Microsoft.Net.Http.Headers.dll
        ObjectCollection`1 in Microsoft.Net.Http.Headers.dll

Microsoft.Win32
        NativeMethods in Microsoft.AspNet.Authentication.dll

Microsoft.Win32.SafeHandles
        SafeHandleZeroOrMinusOneIsInvalid in Microsoft.AspNet.Cryptography.Internal.dll